lib/chrome.js

```diff
@@ -58,6 +59,16 @@ class Events {
 
 const MIN_INTERVAL = 1000;
 
+function keyToId(key) {
+  let md5Str = md5(key);
+  const parts = [];
+  [8,4,4,4,12].map(numChars => {
```

Comment:
This works, but it's a bit silly to parse and then unparse: https://github.com/Kinto/kinto-chrome/commit/958e4296dbd46ae881149d37ccc287859718e5a1